commodore/postprocess/__init__.py

```diff
@@ -8,6 +8,17 @@
 from .builtin_filters import run_builtin_filter
 
 
+def _inventory_resolve(inventory, value):
```

Comment:
Seems kinda wrong to implement this here. But I couldn't find a way to reuse the reclass implementation quickly.

Reply:
I thought about this a bit after seeing the comment. What I've come up with is that we could probably move the whole postprocessing filters definition into the inventory as `parameters.commodore.postprocess.filters` in the component class (similarly to the Kapitan compilation definitions).

That way we'd get already resolved filter arguments etc.

I'd suggest opening an issue for this refactoring, and keep the manual inventory resolving for this PR.

Reply:
After looking at the code again I realized that we already have a partial implementation to resolve references for the builtin filters' `filterargs`. I've refactored the postprocessing code a bit to use this implementation for both filterargs and any other fields in the filter definition dicts.